### PR TITLE
ShowHiddenThings: more effectively explode Algolia filters

### PR DESCRIPTION
**Pull request by dolfies**
Current patch doesn't always work as there's multiple functions for fetching Algolia results, might be related to an experiment

### CODE REVIEW

File: src/plugins/showHiddenThings/index.ts [hunk 1]
```diff
@@ -80,11 +80,19 @@ export default definePlugin({
...
             predicate: () => settings.store.disableDiscoveryFilters,
             replacement: {
-                match: /filters:\i\.join\(" AND "\),facets:\[/,
-                replace: "facets:["
+                match: /\{"auto_removed:.*?\}/,
+                replace: "{}"
+            }
+        },
+        {
+            find: "\">200\"",
```
Comment (Nuckyz) [inline src/plugins/showHiddenThings/index.ts:91]: Patch the getter for MINIMUM_MEMBER_COUNT instead. I know in the past I said it's bad to patch exports, but in this case it's better than targetting a nunber

```suggestion
            find: "MINIMUM_MEMBER_COUNT:",
```

**Review by Nuckyz — CHANGES_REQUESTED** [review]

File: src/plugins/showHiddenThings/index.ts (hunk 1 above)
Comment (dolfies) [reply src/plugins/showHiddenThings/index.ts]: Done!

**Comment by Vendicated** [comment]
thankies

File: src/plugins/showHiddenThings/index.ts (hunk 1 above)
Comment (Vendicated) [reply src/plugins/showHiddenThings/index.ts]: clueless

**Pull request merged**